src/postgres_client.rs

```diff
@@ -52,11 +56,19 @@ struct PostgresSqlClientWrapper {
     update_transaction_log_stmt: Statement,
     update_block_metadata_stmt: Statement,
     insert_account_audit_stmt: Option<Statement>,
+    insert_token_owner_index_stmt: Option<Statement>,
+    insert_token_mint_index_stmt: Option<Statement>,
+    bulk_insert_token_owner_index_stmt: Option<Statement>,
+    bulk_insert_token_mint_index_stmt: Option<Statement>,
 }
 
 pub struct SimplePostgresClient {
     batch_size: usize,
     pending_account_updates: Vec<DbAccountInfo>,
+    index_token_owner: bool,
+    index_token_mint: bool,
+    pending_token_owner_index: Vec<TokenSecondaryIndex>,
+    pending_token_mint_index: Vec<TokenSecondaryIndex>,
     client: Mutex<PostgresSqlClientWrapper>,
```

Comment:
It seems like the inner client is the only thing that needs to be in the mutex, the statements can be extracted outside so we don't need to grab a lock to access the tatements

Reply:
The prepared statements are created with a certain client which is used during prepare-- we cannot use a statement out side of the client creating it. So I think it makes sense putting them together.